src/NetAnalyzers/UnitTests/Microsoft.CodeQuality.Analyzers/QualityGuidelines/MarkMembersAsStaticTests.cs

```diff
@@ -1178,6 +1178,48 @@ End Function
             }.RunAsync();
         }
 
+        [Fact]
+        public async Task AutoProperties_NoDiagnostic()
+        {
+            await VerifyCS.VerifyAnalyzerAsync(@"
+using System.Diagnostics;
+using System.Runtime.CompilerServices;
+
+public class C1
```

Comment:
Do we have a test to verify the following property is flagged:

```csharp
using System.Diagnostics;

public class C1
{
    private static int s_field;
    public int P1
    {
        get { return s_field; }
        set { s_field = value; }
    }
}
```

Reply:
We only have the setter here: https://github.com/dotnet/roslyn-analyzers/blob/master/src/NetAnalyzers/UnitTests/Microsoft.CodeQuality.Analyzers/QualityGuidelines/MarkMembersAsStaticTests.cs#L173